docs/src/routes/+layout.svelte

```diff
@@ -56,6 +57,7 @@
 			<slot />
 		</article>
 	</main>
+	<SvelteToast options={{ reversed: true, intro: { y: 192 } }} />
```

Comment:
Oh yeah thats way better, do you want to keep it inside +layout.svelte or move `--toastBarBackground` to global.css so that all styles for the toast are in the same place?

Reply:
If that works sure! I wasn't sure since that's how we set it in the main App 👀😅